wrappers/services.go

```diff
@@ -1054,9 +1229,11 @@ func ServicesEnableState(s *snap.Info, inter Interacter) (map[string]bool, error
 
 // QueryDisabledServices returns a list of all currently disabled snap services
 // in the snap.
+// XXX: Currently this only retrieves state for the system services, should
+// we extend this to cover all user-services as well?
```

Comment:
these are used by refresh logic, we need to see what are the consequences?

Reply:
I'll remove this XXX: and then do support for user-services in this function as a separate PR, it will need its own tests and everything